pkg/controlsvc/controlsvc.go

```diff
@@ -331,34 +335,37 @@ func (s *Server) RunControlSvc(ctx context.Context, service string, tlscfg *tls.
 					if ctx.Err() != nil {
 						return
 					}
+					if err != nil {
+						if strings.HasSuffix(err.Error(), "normal close") {
```

Comment:
where does this "normal close" come from? can this be done with the `errors.Is` method?